web-src/src/components/NavbarBottom.vue

```diff
@@ -153,15 +156,23 @@
         class="navbar-item px-2"
         :icon_size="24"
       />
+      </group>
+      <group class="navbar-item is-expanded is-justify-content-right is-no-basis">
+      <player-button-lyrics
```

Comment:
Now that I'm looking at the interface, I'm wondering if this button should be placed here. It could be ok for now, but definitely the bottom bar should be redesigned.

I know that there are currently some impracticalities with the volume control as mentioned by @ejurgensen in this [work item](https://github.com/owntone/owntone-server/issues/1442).

I would suggest to mostly copy the design of Spotify or Jellyfin.

Reply:
But this is mostly how [spotify is doing it](https://beebom.com/wp-content/uploads/2021/11/mic-icon-spotify-lyrics.jpg?quality=75&strip=all). The lyrics button is on the bottom right. Jellyfin doesn't seem to support lyrics for music (or I haven't found how on the demo).

Amazon music "hides" the lyrics on the album thumbnails on the bottom left (you click the thumbnail and the lyrics pane appears to replace the complete interface). I don't like this, you can't detect this feature unless someone told you so. When the lyrics are showing, you don't have access to the interface, which is also bad, IMHO. Lyrics shouldn't hide the player information, or you'll spend time toggling the lyrics on & off for each track.

I agree that the (global) volume bar is missing on the bottom bar, and hopefully, my changes to the flex layout now allows to add such widget without breaking the centering of the player's icons. 

So it should be possible to add this after this is merged.

Reply:
Yes, I agree with the points above.
It was more of a general comment for the usability of OwnTone web interface that we'll have to improve in the future.
Let's go ahead with your proposal.

Reply:
Done